scripts/train.py

```diff
@@ -9,41 +7,32 @@
 @click.command()
 @click.option('--loader', '-l', type=click.Choice(['PKLot', 'PKSpace']),
               default='PKSpace', help='Loader used to load dataset')
-@click.option('--model_type', '-mt', type=click.Choice(['MLP']), default='MLP',
+@click.option('--model_type', '-t', type=click.Choice(['MLP']), default='MLP',
               help='Type of model to be trained')
-# @click.option('--hidden_later', '-hl', default=(15, 10),
-#               help='Hidden layers of MLP, if MLP is chosen as model_type')
-@click.option('--model_path', '-pm', type=click.Path(exists=True),
-              default=None,
+@click.option('--hidden_layer', '-h', default=(15, 10), multiple=True,
+              type=int,
+              help='Hidden layers of MLP, if MLP is chosen as model_type')
+@click.option('--model_path', '-p', default=None,
+              type=click.Path(exists=True, dir_okay=False, file_okay=True),
               help='Path to trained model, to be used as a base in training')
-@click.argument('dataset_dir', type=click.Path(exists=True))
-@click.option('--output', '-o', default='out.pkl',
+@click.argument('dataset_dir',
+                type=click.Path(exists=True, file_okay=False, dir_okay=True))
```

Comment:
@rstevanak You should probably also set `resolve_path=True` because of symlinks.